src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -57,6 +69,12 @@ public void bindDrivebaseControls() {
 		driveController.back().onTrue(new InstantCommand(s.drivebaseSubsystem::resetPose));
 	}
 
+	public void bindArmControls() {
+		armManualControl.toggleOnTrue(
+				new ManualArmOverrideCommand(
+						s.armSubsystem, codriveController.getRightY(), codriveController.getLeftY()));
```

Comment:
This will only record the value of the sticks from when the controls are bound, use `DoubleSupplier`s instead.